### DISCUSSION TITLE
Best Model + Best Parameters, reducing lines of code.

### DISCUSSION THREAD

**Discussion by mahnoor-byte** [AammarTufail/python_ka_chilla2023#90] [Q&A] [answered]
I have done this in a very complex way since the examples that used for loops extensively were not making sense to me. Hence I had to do a separate line of code for each model and separate lines for parameters of all models.

1. How can I make this shorter using loops? 
2. I tried adding sorting loop but it didn't work. How to add that as well?

![image](https://user-images.githubusercontent.com/121869407/214376562-2fa36f75-f48b-4172-b6c1-a63e0e44bec0.png)
![image](https://user-images.githubusercontent.com/121869407/214376021-ef448c2e-3581-470f-9b0a-c4d9c9dcf719.png)

**Comment by Muhammad-Ali-Butt** [comment, marked as answer]
PipelineHelper use krna ha

**Reply by Muhammad-Ali-Butt** [reply]
![image](https://user-images.githubusercontent.com/108717967/214652071-2f85d23a-622c-4e82-a8f5-c382c01f44af.png)

**Reply by Muhammad-Ali-Butt** [reply]
btw todo wagera jo likha howa ha wo basically colored comment k liye kia ha aik extension k thru so dont get confused red , blue aur orange colors mai jo sentences likhay hoi ha wo comments ha you can further read about PipelineHelper class here : https://github.com/bmurauer/pipelinehelper

**Reply by mahnoor-byte** [reply]
Got it. Thank you.
